### PR TITLE
refactor: use default plugin blueprint location

### PR DESCRIPTION
**Pull request by soofstad**
## What does this pull request change?
- Update readme to use `dm import-plugin-blueprints node_modules/@development-framework/dm-core-plugins` to import plugin blueprints
- Restructure example blueprints and entities

## Why is this pull request needed?
Follow new convention for plugin blueprints

## Issues related to this change
relates to https://github.com/equinor/dm-cli/issues/64

### CODE REVIEW

File: example/docker-compose.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: "3.4"
 
 services:
   dmss:
-    image: datamodelingtool.azurecr.io/dmss:latest
+    image: datamodelingtool.azurecr.io/dmss:dev
```
Comment (KristianKjerstad) [inline example/docker-compose.yaml:5]: should this be latest?

File: example/README.md [hunk 2]
```diff
@@ -11,6 +11,7 @@
    - `docker-compose run --rm job-api dm -u http://dmss:5000 reset ../app`
 5. Load test app data
     - `dm reset app`
+    - `dm import-plugin-blueprints node_modules/@development-framework/dm-core-plugins`
```
Comment (KristianKjerstad) [inline example/README.md:14]: can also add this to the oneliner command under Development tips

**Review by KristianKjerstad — CHANGES_REQUESTED** [review]

File: example/docker-compose.yaml (hunk 1 above)
Comment (soofstad) [reply example/docker-compose.yaml:5]: fixed

File: example/app/data/DemoDataSource/DemoPackage/entities/Dashboard/productExample.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "_id": "product1",
+  "type": "blueprints/Product",
+  "name": "productExample"
+}
```
[the remark this answers is not in the document]
Comment (soofstad) [reply example/app/data/DemoDataSource/DemoPackage/entities/Dashboard/productExample.json:5]: fixed

File: example/README.md (hunk 2 above)
Comment (soofstad) [reply example/README.md:14]: fixed

**Review by KristianKjerstad — APPROVED** [review]

**Pull request merged**